src/components/views/auth/PasswordLogin.js

```diff
@@ -1,6 +1,6 @@
 /*
 Copyright 2015, 2016 OpenMarket Ltd
-Copyright 2017 Vector Creations Ltd
+Copyright 2017,2019 Vector Creations Ltd
```

Comment:
Seems like this should be The Matrix.org Foundation C.I.C.

Reply:
Changes happened the days before the asset transfer, so are still under NV

Reply:
Well, at least this one should say "New Vector Ltd" then.

Reply:
Oh, yes. That's awkward.